### ISSUE TITLE
train.sh

### ISSUE THREAD

**Issue by November198** [kahnchana/svt#20]
May I ask that the backbone of the reproduction is DINO, so how are the two loss functions proposed in the paper reflected in the code? If MODEL.TWO_STREAM is set to False in the train.sh file, then the two loss functions proposed in the paper are not reflected. Thank you very much for your consideration.

**Comment by kahnchana** [comment]
@Dorothy-h357h thank you for your interest in our work. 

In our code, during training we pass our two global views through teacher ([here](https://github.com/kahnchana/svt/blob/f69e0048f2ed38f5398ee6d500abf50cbb8779f0/train_ssl.py#L511)) and use as targets to guide the student outputs. The self-distillation loss is defined entirely in [loss class](https://github.com/kahnchana/svt/blob/f69e0048f2ed38f5398ee6d500abf50cbb8779f0/train_ssl.py#L586) following the DINO loss (from their paper). 

The two different losses in our paper (local to global and global to global) are implemented together in that single for loop inside the DINO loss class forward function. We generate these multiple views in our dataset class [here](https://github.com/kahnchana/svt/blob/f69e0048f2ed38f5398ee6d500abf50cbb8779f0/datasets/transform.py#L728) and [here](https://github.com/kahnchana/svt/blob/f69e0048f2ed38f5398ee6d500abf50cbb8779f0/datasets/decoder.py#L414). 

Hope this clarifies your question.

**Comment by November198** [comment]
> 



> @Dorothy-h357h thank you for your interest in our work.
> 
> In our code, during training we pass our two global views through teacher ([here](https://github.com/kahnchana/svt/blob/f69e0048f2ed38f5398ee6d500abf50cbb8779f0/train_ssl.py#L511)) and use as targets to guide the student outputs. The self-distillation loss is defined entirely in [loss class](https://github.com/kahnchana/svt/blob/f69e0048f2ed38f5398ee6d500abf50cbb8779f0/train_ssl.py#L586) following the DINO loss (from their paper).
> 
> The two different losses in our paper (local to global and global to global) are implemented together in that single for loop inside the DINO loss class forward function. We generate these multiple views in our dataset class [here](https://github.com/kahnchana/svt/blob/f69e0048f2ed38f5398ee6d500abf50cbb8779f0/datasets/transform.py#L728) and [here](https://github.com/kahnchana/svt/blob/f69e0048f2ed38f5398ee6d500abf50cbb8779f0/datasets/decoder.py#L414).
> 
> Hope this clarifies your question.

Thank you very much! I understand a lot.
